source/CardGames.py

```diff
@@ -374,7 +387,7 @@ def marshall_first_turn(self):
       # Show the cards marshall drew
       self.marshall.showHand()
       # Display the two cards the fugitive has placed, face down
-      self.display_board_general()
+      self.display_board_general(self.cards_in_play)
```

Comment:
This is my fault. We should make sure to remove the parameter in this because the class always has access to self.cards_in_play

Reply:
Got it sounds good!